buildbot/osuosl/master/config/builders.py

```diff
@@ -689,8 +689,8 @@
                         "-DCMAKE_CXX_COMPILER=clang++",
                         "-DCLANG_DEFAULT_LINKER=lld",
                         "-DLLVM_TOOL_GOLD_BUILD=0",
-                        "-DCMAKE_C_COMPILER_EXTERNAL_TOOLCHAIN:PATH=/usr",
-                        "-DCMAKE_CXX_COMPILER_EXTERNAL_TOOLCHAIN:PATH=/usr",
+                        "-DCMAKE_C_COMPILER_EXTERNAL_TOOLCHAIN:PATH=/gcc-toolchain/usr",
+                        "-DCMAKE_CXX_COMPILER_EXTERNAL_TOOLCHAIN:PATH=/gcc-toolchain/usr",
```

Comment:
where will the actual toolchain used be installed?  I think it's best if we have this path to be setup at the root of where all future toolchains are expected to be installed.

Reply:
> where will the actual toolchain used be installed? 

Redhat toolchains usually go under `/opt/rh/` but that is up to `dnf` installation defaults to decide.
We only need to ensure each instance has a symlink from `/gcc-toolchain/usr` to the desired toolchain, right now that is `/opt/rh/gcc-toolset-12/root/`.

>  I think it's best if we have this path to be setup at the root of where all future toolchains are expected to be installed.

This would result in the same problem Galina pointed out via email, if `/gcc-toolchain/` points to `/opt/rh/` instead of `/opt/rh/gcc-toolset-12/root/` then we will have to set `-DCMAKE_C_COMPILER_EXTERNAL_TOOLCHAIN:PATH=/gcc-toolchain/gcc-toolset-12/root/usr` in zorg code again, making toolchain version control more difficult.

Reply:
I was thinking mabye it would be better to link the 2 locations together.  Then it can be seen very clearly that there is a dependency there. ie `/opt/rh/gcc-toolchain->/opt/rh/gcc-toolset-12/root`
Then here we point the toolchain path to `/opt/rh/gcc-toolchain/usr`